.devcontainer/onCreate-conda.sh

```diff
@@ -5,8 +5,9 @@ set -ex
 
 # from https://pyodide.org/en/stable/development/building-from-sources.html#using-make:
 # - build-essential
+# we install file because it is used by packages/ppl during configure
```

Comment:
In this particular case, the configure script appears to look for it specifically in /usr/bin, which is weird and could be patched out of course, but just installing in with apt is the easiest solution. As you say, `file` is a common tool; so it will certainly be present on typical developers' machines.